src/general/OptionsMenu.cs

```diff
@@ -853,6 +856,8 @@ public void ApplySettingsToControls(Settings settings)
         bloomEffectToggle.ButtonPressed = settings.BloomEnabled;
         bloomSlider.Value = settings.BloomStrength;
         bloomSlider.Editable = settings.BloomEnabled || !DisableInactiveSliders;
+        blurSlider.Value = settings.MicrobeBackgroundBlurStrength;
+        blurSlider.Editable = Settings.Instance.MicrobeDistortionStrength > 0 || !DisableInactiveSliders;
```

Comment:
Does the effect look really bad without distortion, or why was this ultimately left so that the blur cannot be enabled unless distortion is also on?

Reply:
It's honestly debatable if blur should only be possible to enable with distortion. My reasoning was that the background were originally not designed to be used with blur, so they might look bad with it.
But if blur is to be applied to them without distortion, it's probably better to just switch the images to pre-blurred ones for more performance. The cost of that would be blur strength being unconfigurable, though.

Reply:
I'd argue the distortion already changes the original artistic vision, and I allowed that so with the same logic blur should be allowed. And as it would be just a tiny bit of code change to not lock out that option while distortion is off, I think it is worth it to allow it. And blurring the background images themselves would need a lot of artistic touch to do it well (it was tried once before and I really disliked the effect), and no one would be able to play with the old backgrounds if they preferred. So all in all I think it is just much superior option to allow to enable the blur separately from the distortion effect.

Reply:
I meant that the pre-blurred images would only be shown when blur is on (otherwise the original backgrounds would remain); though it might feel weird that the slider doesn't control blur strength when distortion is off.

Reply:
I wouldn't want to bloat the game filesize a bunch just to have this feature. So I'm against including both variants of the backgrounds, which is why I think this should be a dynamically rendered effect.